README.md

```diff
@@ -34,6 +34,12 @@ Just run this command on your terminal:
 3.  Open `xcbuddy.xcworkspace`.
 4.  Have fun 🤖
 
+## Guides
+
+[Install](https://github.com/xcbuddy/xcbuddy/blob/master/guides/Install.md)
```

Comment:
Nice 👏 ! I'd use the website link rather than the markdown file ([example](https://xcbuddy.github.io/xcbuddy/error--handling.html))